### DISCUSSION TITLE
video recording

### DISCUSSION THREAD

**Discussion by IndrajeethY** [beeware/beeware#162] [General]
is it possile to record a video using beeware?

**Comment by freakboy3742** [comment]
We don't currently have any cross-platform APIs for capturing camera input.

**Comment by IndrajeethY** [comment]
ohh okie. Is it possible to add my own icon to app?

**Reply by freakboy3742** [reply]
Yes. Each platform has slightly different icon size/shape requirements; but Briefcase has an `icon` attribute you can set to define the icon you want to use for your app. The Briefcase docs have more details.

**Reply by IndrajeethY** [reply]
tqs
